src/main/java/wolox/training/models/User.java

```diff
@@ -22,9 +22,12 @@
 import javax.persistence.JoinColumn;
 import javax.persistence.JoinTable;
 import javax.persistence.ManyToMany;
+import lombok.AccessLevel;
+import lombok.Getter;
 import wolox.training.exceptions.BookAlreadyOwnedException;
 import wolox.training.exceptions.BookNeverOwnedException;
 
+@Getter
 @Entity(name = "USERS")
 public class User {
```

Comment:
You could make use of the @NoArgsConstructor annotation 😃

Reply:
Done! :D